src/main/webapp/script.js

```diff
@@ -12,6 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * Hides reviews when client wantes to see college-info.
+ */
+function hideReviews() {
```

Comment:
I would rename each of these from `hide...` to `show...`, since hiding one doesn't suggest that something else will be shown. On the other hand, it's understood that in order to show something, something else may be hidden. So this would be `showSchoolInfo` and the next would be `showReviews`.